src/model.py

```diff
@@ -49,7 +49,7 @@ def __init__(self):
 
             s3_endpoint_url = os.getenv(
                 "ENDPOINT_URL",
-                "https://rgw-openshift-storage.apps.cnv.massopen.cloud/",
+                "https://rgw-openshift-storage.apps.zero.massopen.cloud/",
```

Comment:
not available

Reply:
The proper endpoint here should be:

https://s3-openshift-storage.apps.zero.massopen.cloud/